spec/rails_helper.rb

```diff
@@ -4,6 +4,7 @@
 require File.expand_path('../../config/environment', __FILE__)
 require 'rspec/rails'
 require 'shoulda-matchers'
+require 'capybara/rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.